src/main/java/com/questhelper/quests/recipefordisaster/RFDEvilDave.java

```diff
@@ -192,7 +193,7 @@ public void setupSteps()
 	{
 		List<PanelDetails> allSteps = new ArrayList<>();
 
-		List<QuestStep> steps = Arrays.asList(inspectEvilDave, enterBasement,
+		List<QuestStep> steps = QuestUtil.toArrayList(inspectEvilDave, enterBasement,
```

Comment:
Unfortunately, `Arrays.asList` doesn't return the `ArrayList` in `java.util.ArrayList`. It returns a fixed-size list so we can't add or remove anything from the list.